### PR TITLE
website_creation_howard

### PR DESCRIPTION
**Pull request by howardjian**
howard’s version of website creation

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<html>
+  <head>
+    <h1>Big Ass Title </h1>
+  </head>
+
```
Comment (jurytan) [inline index.html]: Let's remove this space

File: index.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  <p>
+    This is a paragraph showing I can have
+  </p>
+
```
Comment (jurytan) [inline index.html]: remove this space

File: index.html [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+<h2>
+  List of Things I like
+</h2>
+  <ul>
+    <li>Butts </li>
```
Comment (jurytan) [inline index.html]: Let's use Ass instead of Butts

**Review by jurytan — CHANGES_REQUESTED** [review]
Please review the comments and make the changes before we merge this.

File: index.html (hunk 1 above)
Comment (howardjian) [reply index.html]: Done

File: index.html (hunk 2 above)
Comment (howardjian) [reply index.html]: done

File: index.html (hunk 3 above)
Comment (howardjian) [reply index.html]: done

**Comment by howardjian** [comment]
@jnguyen7410 made changes, please review

**Review by jurytan — APPROVED** [review]
Looks good

**Pull request merged**